### ISSUE TITLE
Dragging the Form gets interrupted when hiding the another form in RemoteApp environment

### ISSUE THREAD

**Issue by GokulThanu** [dotnet/winforms#9772]
### .NET version

The project is tagerting .netframework 4.6. This issue is also reproduced in .net6.0

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No.

### Issue description

Dragging the Form gets interrupted when hiding the another form in RemoteApp environment

Without using the remoteapp environment it was working properly.

### Steps to reproduce

[Dragging of the form gets interrupted.zip](https://github.com/dotnet/winforms/files/12419167/Dragging.of.the.form.gets.interrupted.zip)

- Run the sample
-click on the button

- Drag the opened form to the right.
- Note that the dragging was interupted when hiding another form

**Comment by JeremyKuhne** [comment]
@GokulThanu I'm not seeing this issue when running in a VM with just normal remoting. Are you talking about this service? https://learn.microsoft.com/en-us/azure/virtual-desktop/remote-app-streaming/overview?

It wouldn't surprise me that this is some quirk related to remoting just the application window. I suspect that there may be nothing we can do here.

**Comment by GokulThanu** [comment]
Hi @JeremyKuhne ,

The Remote App environment that i am referring is environment where we can access the publised applications alone. Please refer the below link for you reference.

https://youtu.be/HRIQ44t1QZU?si=H9nJ77GY3qIpCTpL

Please refer the attached video reference where you can find dragging the form gets interrupted when i hide the red color form.

https://github.com/dotnet/winforms/assets/111874891/ebbd88ff-7de1-4cb1-ae2b-08e700aad8e5

**Comment by elachlan** [comment]
@Olina-Zhang could your team test this please?

**Comment by Olina-Zhang** [comment]
@GokulThanu, I configured a server machine, and publish VisualStudio in Remote Desktop Service, and test that application you provided with different .Net framework versions and .Net versions to .Net 8 in another machine by remoteApp connecting , they all have same testing result as yours. But I saw the child form doesn't get interrupted fully, it can also be dragged to move step by step, just not smooth. Right?

![RemoteAppTesting](https://github.com/dotnet/winforms/assets/26474449/abef3eb5-4ae8-4fc4-99a9-ea98e87a4846)

**Comment by GokulThanu** [comment]
@Olina-Zhang  yes once i hide the other form(red in color) the dragging form gets interrupted. if i am not hiding the form i can drag the form smoothly

**Comment by GokulThanu** [comment]
@Olina-Zhang yes. Before hiding the form(red in color) able to drag the dragging form smoothly, but when i hide the form(in red color) the dragging form gets interruppted fully.
